.travis.yml

```diff
@@ -15,6 +15,11 @@ matrix:
   allow_failures:
     - go: master
 
+# skip non-trunk PMM-XXXX branch builds, but still build pull requests
+branches:
+  except:
+    - /^PMM\-\d{4}/
+
 go_import_path: github.com/prometheus/node_exporter
```

Comment:
Is this path correct?

Reply:
Yes, we use that import path in our fork.